web_src/css/repo.css

```diff
@@ -1547,14 +1552,21 @@ td .commit-summary {
 .labels-list {
   display: inline-flex;
   flex-wrap: wrap;
-  gap: 2.5px;
-  align-items: center;
+  overflow: hidden;
+  gap: 0.25rem;
+  line-height: 1;
```

Comment:
No, we should never use line-height=1, it will cause problem with overflow hidden, some parts for letters like `g` will be hidden.

Reply:
Hmm, but how else to hide the extra space. if you set `gap: 0`, there should be zero space between lines but isn't without this.

Reply:
By setting a smaller vertical gap, it looks alright now:

<img width="347" alt="image" src="https://github.com/user-attachments/assets/bd90686d-09d8-4114-86d3-429e74f57254" />